server/graphql/resolvers.ts

```diff
@@ -27,6 +28,52 @@ const getRequestsById = (requestIds, dataSources) => {
   return requestIds.map(id => dataSources.requests.getById(id))
 }
 
+const updateRequestGroupHelper = (requestGroup, dataSources, dateUpdated = Date.now()): Promise<ServerResponseInterface> => {
+  requestGroup.dateUpdated = dateUpdated;
+  return dataSources.requestGroups.update(requestGroup)
+    .then(res => {
+      return res
+    })
+    .catch(error => {
+      return error
+    })
+}
+
+const updateRequestTypeHelper = (requestType, dataSources, dateUpdated = Date.now()): Promise<ServerResponseInterface> => {
+  if(!requestType.id) {
+    throw new UserInputError('Missing argument value', { argumentName: 'id' })
+  }
+  const requestGroupId = dataSources.requestTypes.getById(requestType.id).requestGroup
+  requestType.dateUpdated = dateUpdated
+  return dataSources.requestTypes.update(requestType)
+    .then(res => {
+      const requestGroup = dataSources.requestGroups.getById(requestGroupId.toString())
+      updateRequestGroupHelper(requestGroup, dataSources, requestType.dateUpdated)
```

Comment:
Are we planning on changing the way `CachedMongooseDataSource.getById` and `CachedMongooseDataSource.update` report errors?

If yes, I'm OK with how there's no extensive error checking here right now